kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -5,14 +5,16 @@
 public class FlightFinder {
 
     private final Map<String, Boolean> mapOfFlights;
+    private static final String ERROR_COMMUNICATE = "There is no flight from %s Airport to %s Airport check if you get "
+            .concat("correct departure point or contact with our call center (+48 121 12 12)");
```

Comment:
w takich przypadkach nie stosuj concat. Poprostu napisza wsyzstko w jednej linijce, a potem naciśnij enter tam, gdzie chcesz złamać linijkę, a intellij idea sam to za ciebie zrobi (i tu dochodzimy do sprzeczności, że będzie użyty znak "+" na konkatenację i ja mówię, że tak będzie lepiej :-) pogadamy na spotkaniu o tym :-)